.github/ISSUE_TEMPLATE/Bug_issue.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Report a bug
+about: Report an issue that you have encountered.
+title: ''
+labels: 'Type: bug'
+---
+
+#### Summary
```

Comment:
- missing `import cil, sys; print(cil.version.commmit_hash, sys.version, sys.platform)`
- should open a follow-up issue to have `cil.version.__str__` to return all of the above to make life easier :)

Reply:
I added a sentence about the cil version. not looking great

Reply:
see below, I would add the comment fully rendered, not in a comment. The user might read it!